php/WP_CLI/Runner.php

```diff
@@ -966,7 +972,8 @@ private function check_wp_version() {
 				);
 			}
 			WP_CLI::error(
-				"This does not seem to be a WordPress installation.\n" .
+				"The path does not seem to be a WordPress installation.\n" .
```

Comment:
```suggestion
				"This path does not seem to be a WordPress installation.\n" .
```

If we restore the original string, we can probably avoid breaking/changing a lot of test assertions.